src/selectors/entities/channels.test.js

```diff
@@ -9,6 +9,884 @@ import {sortChannelsByDisplayName, getDirectChannelName} from 'utils/channel_uti
+        channelsInTeam[team2.id] = [channel2.id];
+        channelsInTeam[''] = [channel4.id];
+
+        const testState = deepFreezeAndThrowOnMutation({
+            entities: {
+                users: {
+                    currentUserId: user.id,
+                    profiles,
+                },
+                teams: {
+                    currentTeamId: team1.id,
+                },
+                channels: {
+                    channels,
+                    channelsInTeam,
+                },
+            },
+        });
+
+        const channelsInCurrentTeam = [channel1, channel3].sort(sortChannelsByDisplayName.bind(null, []));
+        assert.deepEqual(Selectors.getChannelsInCurrentTeam(testState), channelsInCurrentTeam);
+    });
+
+    it('should order by user locale', () => {
+        const userDe = TestHelper.fakeUserWithId();
+        userDe.locale = 'de'
+
+        const userSv = TestHelper.fakeUserWithId();
+        userSv.locale = 'sv'
+
+        const profilesDe = {};
+        profilesDe[userDe.id] = userDe;
+
+        const profilesSv = {};
+        profilesSv[userSv.id] = userSv;
+
+        const channel1 = TestHelper.fakeChannelWithId(team1.id);
+        channel1.display_name = 'z';
+        const channel2 = TestHelper.fakeChannelWithId(team1.id);
+        channel2.display_name = 'ä';
+
+        const channels = {};
+        channels[channel1.id] = channel1;
+        channels[channel2.id] = channel2;
+
+        const channelsInTeam = {};
+        channelsInTeam[team1.id] = [channel1.id, channel2.id];
+
+        const testStateDe = deepFreezeAndThrowOnMutation({
+            entities: {
+                users: {
+                    currentUserId: userDe.id,
+                    profiles: profilesDe,
+                },
+                teams: {
+                    currentTeamId: team1.id,
+                },
+                channels: {
+                    channels,
+                    channelsInTeam,
+                },
+            },
+        });
+
+        const testStateSv = deepFreezeAndThrowOnMutation({
+            entities: {
+                users: {
+                    currentUserId: userSv.id,
+                    profiles: profilesSv,
+                },
+                teams: {
+                    currentTeamId: team1.id,
+                },
+                channels: {
+                    channels,
+                    channelsInTeam,
+                },
+            },
+        });
+
+        const channelsInCurrentTeamDe = [channel1, channel2].sort(sortChannelsByDisplayName.bind(null, userDe.locale));
+        const channelsInCurrentTeamSv = [channel1, channel2].sort(sortChannelsByDisplayName.bind(null, userSv.locale));
+
+        assert.deepEqual(Selectors.getChannelsInCurrentTeam(testStateDe), channelsInCurrentTeamDe);
+        assert.deepEqual(Selectors.getChannelsInCurrentTeam(testStateSv), channelsInCurrentTeamSv);
+    });
+});
+
+describe('Selectos.Channels.getMyChannels', () => {
```

Comment:
```suggestion
describe('Selectors.Channels.getMyChannels', () => {
```
Although I think you could also consider dropping the `Selectors.Channels` part of the test suite name anyway since that's implied by the location of the file anyway.

Reply:
If we remove Selectors because is implied by the file, shall we drop also Channels?

Reply:
Yeah, I'd lean towards that myself. Feel free to leave it as-is though since we aren't really consistent with that naming.